pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1470,10 +1470,14 @@ self: super: {
   pandoc-types = self.pandoc-types_1_21;
   rfc5051 = self.rfc5051_0_2;
 
+  stylish-haskell = appendPatch super.stylish-haskell (pkgs.fetchpatch {
+    url = "https://github.com/jaspervdj/stylish-haskell/commit/9550aa1cd177aa6fe271d075177109d66a79e67f.patch";
+    sha256 = "159jr80k40hdq5gpqfjknqx6vj2licx1l0f57l5r3k4264lnxjdb";
+  });
```

Comment:
Could you add a comment in the code here why this patch is needed, link to an upstream PR that contains this commit (if it exists), and maybe add when you think this patch can be removed?